### ISSUE TITLE
Formatting not working after the value prop is added

### ISSUE THREAD

**Issue by everdrone** [medipass/react-payment-inputs#13]
Just as described in the redme, placed the event handler inside the getter props but formatting is not applied.

```jsx
import React, { useState } from "react";
import { usePaymentInputs } from "react-payment-inputs";

export default function PaymentInputs() {
  const [number, setNumber] = useState("");
  const [expiry, setExpiry] = useState("");
  const [cvc, setCvc] = useState("");

  const {
    meta,
    getCardNumberProps,
    getExpiryDateProps,
    getCVCProps
  } = usePaymentInputs();

  return (
    <div>
      <div>
        <input
          {...getCardNumberProps({
            onChange: e => setNumber(e.target.value)
          })}
          value={number}
        />
      </div>
      <div>
        <input
          {...getExpiryDateProps({
            onChange: e => setExpiry(e.target.value)
          })}
          value={expiry}
        />
      </div>
      <div>
        <input
          {...getCVCProps({ onChange: e => setCvc(e.target.value) })}
          value={cvc}
        />
      </div>
      {meta.isTouched && meta.error && <div>Error: {meta.error}</div>}
      <button>place order</button>
    </div>
  );
}
```

what am i doing wrong?

**Comment by bartoszhernas** [comment]
I have the same issue. I see that in the package there is `utils/formatter/formatCardNumber` but I cannot import it. Can this function be made public?

**Comment by edmond-c** [comment]
The `onChange` prop is simply forwarded as shown below. Perhaps a new prop `onFormattedValue(value)` would be useful

https://github.com/medipass/react-payment-inputs/blob/master/src/usePaymentInputs.js#L173-L191

**Comment by bartoszhernas** [comment]
expiryDate is formatted correctly, the only problem is with cardNumber 

I've fixed it by copying the `formatCardNumber` code into my project, and then using it when passing down the value.

```
        value={formatCardNumber(cardNumber || "")}
```

Also another bug is that this function requires sting, null or undefined will raise an exception.

**Comment by jxom** [comment]
Should be fixed in `1.0.7`. Let me know how it goes!

**Comment by bartoszhernas** [comment]
Both issues are indeed resolved, thanks :)
